docs/config-file-ort-yml.md

```diff
@@ -55,7 +55,7 @@ excludes:
   paths:
   - pattern: "A glob pattern matching files or paths."
     reason: "One of PathExcludeReason e.g. BUILD_TOOL_OF or DOCUMENTATION_OF."
-    comment: "A comment further explaining why the exclude ."
+    comment: "A comment further explaining why the path is excluded."
```

Comment:
Regarding `why the path is excluded` often times is obvious, e.g. because it is not used, sometimes its not obvious though. I found that in many cases it makes sense to make the comment just explain `what it is` e.g. `what does the path matcher actually match`. Be it a `tests` or `examples` or `build scripts`.